release-notes/10.0/preview/preview1/dotnetmaui.md

```diff
@@ -1,15 +1,150 @@
+  - [Building with JDK-21 is now supported](#building-with-jdk-21-is-now-supported)
+  - [`dotnet run` support for Androd projects](#support-for-dotnet-run-for-android)
+  - [Enable marshal methods by default](#enable-marshal-methods-by-default)
+  - [Visual Studio Design-Time Builds no longer invoke `aapt2`](#visual-studio-design-time-builds-no-longer-invoke-aapt2)
+- [.NET for iOS, Mac Catalyst, macOS, tvOS](#net-for-ios-mac-catalyst-macos-tvos)
+  - [Trimmer warnings enabled by default](#trimmer-warnings-enabled-by-default)
+  - [Bundling original resources in libraries](#bundling-original-resources-in-libraries)
 
 .NET MAUI updates in .NET 10:
 
 - [What's new in .NET MAUI in .NET 10](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-10) documentation.
 
+> **Known Issue:** building `net8.0-*` target frameworks with the .NET 10 SDK does not work in Preview 1.
 
-## Feature
+## CollectionView enhancements for iOS and Mac Catalyst
 
-This is something about the feature
+Two new handlers for `CollectionView` and `CarouselView` on iOS and Mac Catalyst that brought performance and stability improvements were available optionally in [.NET 9](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-9?view=net-maui-9.0#collectionview-and-carouselview). In this release they are now on by default. 
+
+You can _opt out_ of these handlers by adding the following code to your `MauiProgram` class if you want to revert back.
+
+```csharp
+#if IOS || MACCATALYST
+builder.ConfigureMauiHandlers(handlers =>
+{
+    handlers.AddHandler<Microsoft.Maui.Controls.CollectionView, Microsoft.Maui.Controls.Handlers.Items.CollectionViewHandler>();
+    handlers.AddHandler<Microsoft.Maui.Controls.CarouselView, Microsoft.Maui.Controls.Handlers.Items.CarouselViewHandler>();
+});
+#endif
+```
+
+We are excited for you to give these new handlers a try. As a reminder, if you have .NET 9 applications you can try them today with the following code:
+
+```csharp
+#if IOS || MACCATALYST
+builder.ConfigureMauiHandlers(handlers =>
+{
+    handlers.AddHandler<Microsoft.Maui.Controls.CollectionView, Microsoft.Maui.Controls.Handlers.Items2.CollectionViewHandler2>();
+    handlers.AddHandler<Microsoft.Maui.Controls.CarouselView, Microsoft.Maui.Controls.Handlers.Items2.CarouselViewHandler2>();
+});
+#endif
+```
+
+## .NET for Android
+
+This release was focused on quality improvements. A detailed list can be found on [dotnet/android GitHub releases](https://github.com/dotnet/android/releases/).
+
+### Android 16 (Baklava) Beta 1
+
+Google has released [Beta 1](https://android-developers.googleblog.com/2025/01/first-beta-android16.html) of the Android 16 (API-36) SDK.  Support has been added for using these preview APIs.
+
+To target the Android 16 preview API:
+- Use the Android SDK Manager to download the Android 16 (Baklava) Platform
+- Update your project's `TargetFramework` to `net10.0-android36`
+
+### Updated Recommended Minimum Supported Android API

```

Comment:
Is something needed/recommended here for existing projects? Can we add info here on what people should do?